src/libstd/primitive_docs.rs

```diff
@@ -413,6 +415,8 @@ mod prim_isize { }
 //
 /// The pointer-sized unsigned integer type.
 ///
+/// `usize` is guaranteed to be at least as large as `u16`.
```

Comment:
I don't think this change to the language (and therefore RFC) is required. If we ever support a platform with 8-bit pointers (which is unlikely, but still) it would be much easier to just insert a `cfg` before `impl_from! { u16, usize }`.

For reference:
>For the smallest AVR devices with less than 256 bytes of SRAM the tiny memory model
can be used in many cases. When using tiny memory model, all variables in SRAM are
accessed with 8-bit pointers instead of 16-bit pointers. This reduces the code size for
loading pointer values.

http://www.atmel.com/images/doc1497.pdf